codestodians/komlie-frontend
# [Feature] Initialize cart on authentication

#### What does this PR do?
- Initializes the cart details on user authentication
- Updates cart items when mutations are carried out
- Bump styling of cart page

#### Description of Task to be completed?
- Query for user cart items on authentication
- Update styles for cart page
- Refactor mutation updates

#### How should this be manually tested?
- Start the app, get authenticated, add or remove products from cart to see results

#### What are the relevant pivotal tracker stories?
- [#167107452](https://www.pivotaltracker.com/story/show/167107452)
